README.md

```diff
@@ -1 +1,85 @@
-# Awesome-Book-ES6
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Awesome Book - ES6</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
```

Comment:
- [ ] These sections(**Future Features, Key features**) from this Microverse [README](https://github.com/microverseinc/readme-template/edit/master/README.md) template are still missing in your README.md file. These are very important sections that should be present in your README file to render it more descriptive and professional. Kindly add them.